yarn.lock

```diff
@@ -21739,6 +21739,18 @@ object-path-immutable@^3.1.1:
   dependencies:
     is-plain-object "3.0.0"
 
+object-prototype-functions@^1.2.0:
+  version "1.2.0"
+  resolved "https://registry.yarnpkg.com/object-prototype-functions/-/object-prototype-functions-1.2.0.tgz#59ffa520f2e7ef3b3ba427ba81178eb8feca1048"
+  integrity sha512-7pfkHi5KHFz/B3RhQVORykDYF2CdQY/h1isJzCJJfEubKkSO59io1raYHAMUrx0Im3fUSrU1oX6E4KeCwyU/jQ==
+
+object-prototype@^2.0.0:
```

Comment:
I left out `__proto__` and `constructor` as these are properties and not functions. And in the case of a prototype-less object, these are both `undefined` so it should behave as expected in that case.

The `__noSuchMethod__` function isn't in Chrome/Node.js. I think there's an error on the page you linked to as the `__noSuchMethod__` function doesn't have a trashcan next to it even though it was removed in Gecko 43 (and also in Chrome and Node.js). On [this MDN page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/noSuchMethod) it is correctly listed as obsolete.

Reply:
I guess technically on `Object.prototype` that `__proto__` is `null` and `constructor` is undefined. So to 100% mimic that, it might make sense to define them with those values 🤔

Reply:
While making other changes I actually updated the library to ensure it mimics the behavior of `__proto__` and `Object.getPrototypeOf`. The `constructor` property is a different matter unfortunately and can't easily be faked. So I've left that `undefined` still. I don't think that's an issue.